### PR TITLE
Show radius for a few objects in the editor

### LINKED ISSUE #2677: The radius image of a flame is static.
Instead of it being an image with pre-determined height and width, it should be calculated in code to showcase the actual radius of a flame.

### PR DESCRIPTION
**Pull request by MatusGuy**
This pull request resizes the flame radius image in the editor to the specified radius and adds radius indicator for the circle platform.

Fixes #2677

### CODE REVIEW

File: src/badguy/flame.cpp [hunk 1]
```diff
@@ -127,7 +130,21 @@ Flame::active_update(float dt_sec)
     m_sprite->set_angle(math::degrees(angle) * 3.0f);
 
   if (m_sprite->get_action() == "fade" && m_sprite->animation_done())
-    remove_me();
+  remove_me();
```
Comment (Vankata453) [inline src/badguy/flame.cpp:133]: Revert the tabulation here.

File: src/badguy/flame.cpp [hunk 2]
```diff
@@ -37,6 +38,8 @@ Flame::Flame(const ReaderMapping& reader, int type) :
   speed(),
   sound_source()
 {
+  m_range_indicator = Surface::from_file("images/creatures/flame/flame-editor.png");
```
Comment (Vankata453) [inline src/badguy/flame.cpp:41]: Initialize this in the initialization list above.

File: src/object/circleplatform.cpp [hunk 3]
```diff
@@ -87,6 +90,19 @@ CirclePlatform::on_flip(float height)
   FlipLevelTransformer::transform_flip(m_flip);
 }
 
+void CirclePlatform::draw(DrawingContext& context)
```
Comment (Vankata453) [inline src/object/circleplatform.cpp:93]: `void` on a separate line.

File: src/object/circleplatform.cpp [hunk 4]
```diff
@@ -30,6 +31,8 @@ CirclePlatform::CirclePlatform(const ReaderMapping& reader) :
   timer(),
   time(0.0)
 {
+  m_range_indicator = Surface::from_file("images/objects/platforms/circleplatform-editor.png");
```
Comment (Vankata453) [inline src/object/circleplatform.cpp:34]: Initialize this in the initialization list above.

File: src/object/player.cpp [hunk 5]
```diff
@@ -2504,7 +2504,7 @@ Player::set_ghost_mode(bool enable)
 void
 Player::start_climbing(Climbable& climbable)
 {
-  if (m_climbing || m_swimming)
+  if (m_climbing || m_swimming || m_stone)
```
Comment (Vankata453) [inline src/object/player.cpp:2507]: Is this related to this PR?

Comment (MatusGuy) [reply src/object/player.cpp:2507]: Oh my goodness no it isn't... how did that get in here?

**Review by Vankata453 — APPROVED** [review]
